stumpy/core.py

```diff
@@ -1715,12 +1715,10 @@ def preprocess_diagonal(T, m):
     subsequence using awindow size of `m` and `m-1`, respectively, and the inverse
```

Comment:
We may say:
"Creates a copy of the time series where all NaN and inf values are replaced with zero. **THEN,** computes means, `M_T` and `M_T_m_1`, for every subsequence using window size of `m` and `m-1`, respectively"